.allow_skipping_tests

```diff
@@ -15,6 +15,7 @@ controllers/case_contact_reports_controller.rb
 controllers/case_contacts/followups_controller.rb
 controllers/case_court_orders_controller.rb
 controllers/case_court_reports_controller.rb
+controllers/checklist_items_controller.rb
```

Comment:
We will need to fix every line in this file someday, I'll generate the issues. We should stop adding to it where we can...